docs/source/metoffice.rst

```diff
@@ -0,0 +1,204 @@
+.. _MetOfficeUsage:
+
+Using Fab at the Met Office
+===========================
+
+For users working on Met Office desktop systems the following instructions may
+prove useful.
+
+Using Fab
+~~~~~~~~~
+
+If you just want to use Fab to build your source then there are a number of
+fairly simple approaches.
+
+
+Fortran Only
+------------
+
+If you don't need to build C, you can use this the modules in the "extra
+software" collection:
+
+.. code-block:: console
+
+    $ module use $EXTRA_SOFTWARE_MODULE_DIR
+    $ module load python/3.7.0 support/fparser/0.0.16/python/3.7.0 fab/1.0.0/python/3.7.0
+
+The Science I.T. group can provide the `EXTRA_SOFTWARE` path.
```

Comment:
Now that I can successfully load the correct modules, is the next step then to simply install Fab? If so, I think a quick "Then [install fab](https://metomi.github.io/fab/install.html#install)" at the end of the **Fortran only** section might be useful here for new users, as I wasn't too sure which step to follow next (if I don't need to build C).

Reply:
I've added a sentence to make explicit what loading the module achieves.